src/app/donation-start/donation-start.component.html

```diff
@@ -179,7 +179,7 @@ <h1 class="b-rh-1 b-bold">{{ campaign.title }}</h1>
             <p class="b-rt-1 b-bold" id="optInCharityEmail-label">Would you be happy to receive emails from {{ campaign.charity.name }}?</p>
             <mat-radio-group tabindex="0" aria-labelledby="optInCharityEmail-label" formControlName="optInCharityEmail">
               <!-- TODO update this hint to vary based on Gift Aid selection and be clearer -->
-              <mat-hint>This may include thanks for your donation, updates on how it is being used, newsletters and future campaigns. If you select no, we will pass on your details as these may be required for Gift Aid purposes, but the charity will be informed that you do not wish to receive communications.</mat-hint>
+              <mat-hint>{{ campaign.charity.charityOptInStatement }}</mat-hint>
```

Comment:
We should have a default, at least until this is fully rolled out on both apps so we don't have to release in a specific order